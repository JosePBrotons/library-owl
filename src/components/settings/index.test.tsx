import React from 'react';
import renderer from 'react-test-renderer';
import Settings from '.';

test('renders settings component correctly', () => {
    const settingsCmp = renderer.create(<Settings />).toJSON();
    expect(settingsCmp).toMatchSnapshot();
});
