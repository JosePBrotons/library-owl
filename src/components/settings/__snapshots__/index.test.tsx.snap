// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders settings component correctly 1`] = `
<View>
  <Text>
    Hola settings
  </Text>
</View>
`;
